Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -201,6 +201,21 @@ for a detailed implementation example.
     password validators. It can also be used as a resource for writing your
     own password validator.
 
+Validate a password manually
+============================
+
+You can use the :php:`\TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyValidator` to validate a
```

Comment:
Can you give a scenario where a user has to deal with it? I see a command as example, but what is the problem it solves?

Reply:
In my case I was generating a public/private key pair where the private key is secured by a password. But the scope of this example is only to show how you can use the password validation.

Reply:
I would like to give an example to inspire people how to use it.